src/component/parser/json/converter/utils.ts

```diff
@@ -116,4 +117,24 @@ export class ConvertedElements {
   registerGlobalTask(id: string, kind: GlobalTaskKind): void {
     this.globalTasks.set(id, kind);
   }
+
+  private categoryValues: Map<string, CategoryValueData> = new Map();
+  registerCategoryValues(id: string, value: string): void {
+    this.categoryValues.set(id, { value });
+  }
+
+  // Special case: create the ShapeBpmnElement instance here to avoid duplication in CollaborationConverter and CollaborationConverter
+  toGroupShapeBpmnElement(groupBpmnElement: TGroup, processId?: string): ShapeBpmnElement | undefined {
```

Comment:
To keep consistency, I prefer to put this method in the ProcessConverter or to keep in the `utils.ts` file, but not in the `ConvertedElements` class (it's not a related to the storage of converted elements), juste as utils function; and rename it with `buildShapeBpmnGroup`.

Reply:
I put the function in `ConvertedElements` for the following reasons:
- The function must have access to an instance of `ConvertedElements` which is created each time we start a parsing
- We cannot put the function in `ProcessConverter`, otherwise, we should inject a `ProcessConverter` in the class  `CollaborationConverter`.
- So, this function should be put in a dedicated class that would be injected a `ConvertedElements`, and the new class should be injected in both `ProcessConverter` and `CollaborationConverter.
- This seemed to me something too complex for just an object instanciation, so I decided to put the function here

For the function rename, no problem

Reply:
function rename done in 1b97b04